src/color/rgb.js

```diff
@@ -86,7 +86,9 @@ function d3_rgb_parse(format, rgb, hsl) {
   }
 
   /* Named colors. */
-  if (color = d3_rgb_names.get(format)) return rgb(color.r, color.g, color.b);
+  if (typeof format === "string") {
```

Comment:
The type of `format` is guaranteed to be a string here, so this check is redundant.